src/main/java/com/intel/chimera/utils/Utils.java

```diff
@@ -135,8 +135,9 @@ public static int getBufferSize(Properties props) {
    */
   public static String getCipherClassString(Properties props) {
     final String configName = CHIMERA_CRYPTO_CIPHER_CLASSES_KEY;
-    return props.getProperty(configName) != null ? props.getProperty(configName) : System
-        .getProperty(configName, CHIMERA_CRYPTO_CIPHER_CLASSES_DEFAULT);
+    return props.getProperty(configName) != null ? props
```

Comment:
If we do want to handle the case that if am empty class string is specified, we want to default to OpensslCipher,
We can check empty string here and set to CLASSES_DEFAULT for empty value.